pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -61,6 +62,7 @@ func NewDashboardFileReader(cfg *DashboardsAsConfig, log log.Logger) (*fileReade
 	return &fileReader{
 		Cfg:              cfg,
 		Path:             path,
+		cfgPath:          path,
```

Comment:
I'm not sure I like this. seems confusing to have two path properties. How about passing Path as a parameter to `startWalkingDisk` ?

Reply:
Yes make sense Ill add changes.